src/data/CartService.tsx

```diff
@@ -0,0 +1,47 @@
+import CartProduct from "../domain/CartProduct";
+import axios from 'axios';
+import Product from "../domain/Product";
+
+
+const API = 'http://localhost:4000'
+let products: CartProduct[] = []
+
+
+export function addProductInUserCart(product: Product){
+    const cartProduct = {
+        'productId': product.id, 
+        'name': product.name, 
+        'price': product.price, 
+        'quantity': 1} as CartProduct
+    products.push(cartProduct)
+}
+
+export function updateProductFromUserCart(newProduct: CartProduct){
+    const oldProductIndex = products.findIndex(prod => prod.productId === newProduct.productId)
+    products[oldProductIndex] = newProduct
+}
+
+
+export function getProductsFromUserCart(){
+    return products
+}
+
+
+export function isProductsInUserCart(){
+    return products.length > 0;
+}
+
+export function clearUserCart(){
+    products = []
+}
+
+export function sendUserOrder(){
+    const productToSend = products.map(product => { return {'productId': product.productId, 'quantity': product.quantity}})
+    const data = {
+        'customer': 'doej', 
+        'products': productToSend
+    }
+    console.log(data)
```

Comment:
Please avoid leaving logs in your commits. Logs should be only used for debugging purposes only. (Albeit, you may use them if you wish to display errors. You could write an error handler to prettify the error stack.)